Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CartesianCKCAlgorithm.H

```diff
@@ -252,6 +275,8 @@ struct CartesianCKCAlgorithm {
         return alphaz * (F(i  ,j+1,k  ,ncomp) - F(i  ,j  ,k  ,ncomp))
              + betazx * (F(i+1,j+1,k  ,ncomp) - F(i+1,j  ,k  ,ncomp)
                       +  F(i-1,j+1,k  ,ncomp) - F(i-1,j  ,k  ,ncomp));
+#elif (defined WARPX_DIM_1D_Z)
+        return alphaz * (F(i  ,j+1,k  ,ncomp) - F(i  ,j  ,k  ,ncomp));
```

Comment:
Am I correct that this should use `i+1` here?
```suggestion
        return alphaz * (F(i+1,j  ,k  ,ncomp) - F(i  ,j  ,k  ,ncomp));
```

Reply:
Yes, that's correct. Thanks for catching that!